tests/test_patches.py

```diff
@@ -54,22 +59,64 @@
 
 # If you're changing these, most likely need to update setup.py as well.
 _API_DIGESTS = {
-    ClientArgsCreator: {'917d7f5a85d2497c0ee0d7f368a4e6a723bb534a'},
+    # args.py
+    ClientArgsCreator.get_client_args: {'e3a44e6f50159e8e31c3d76f5e8a1110dda495fa'},
+
+    # client.py
     ClientCreator._create_client_class: {'5e493d069eedbf314e40e12a7886bbdbcf194335'},
     ClientCreator._get_client_args: {'555e1e41f93df7558c8305a60466681e3a267ef3'},
-    BaseClient: {'d2e69f0184ae83df5b82284194ed1b72191f0161'},
+
+    BaseClient._make_api_call: {'0c59329d4c8a55b88250b512b5e69239c42246fb'},
+    BaseClient._make_request: {'033a386f7d1025522bea7f2bbca85edc5c8aafd2'},
+    BaseClient.get_paginator: {'c69885f5f73fae048c0b93b43bbfcd1f9c6168b8'},
```

Comment:
Yes, narrows down the diffing :smile:

Reply:
actually I was thinking we can probably do this automatically via inspection